OpenRA.Game/Network/OrderIO.cs

```diff
@@ -109,6 +109,27 @@ public static bool TryParseSync(byte[] packet, out (int Frame, int SyncHash, ulo
 			return true;
 		}
 
+		public static bool TryParsePing(int fromClient, byte[] packet, out byte[] ping)
+		{
+			// Valid Ping packets are only ever generated by the server
+			if (fromClient != 0 || packet.Length != 13 || packet[4] != (byte)OrderType.Ping)
```

Comment:
If the client is supposed to treat this as opaque blob, then why not simply accept any length here?

Reply:
The idea is to minimize the chance of falsely matching random bytes as a valid packet so that we can fail correctly if something goes wonky in the network stream sending/parsing - this is already a problem with order packets (which we plan on fixing in a future PR), which caused me several headaches across this PR chain.